index.html

```diff
@@ -45,30 +45,76 @@ <h1>{A little bit about me...}</h1>
 				<p>Please keep scrolling to see some of my work.</p>
 			</div>
 		</section>
+
+		<section id="portfolio" class="portfolio">
+			<div class="container">
+				<h1>{My Portfolio}</h1>
+					<div class="row">
+					  <div class="column">
+					    <div class="content">
+					      <img src="Images/pholder.jpg" alt="placeholder" style="width: 100%">
+					      <h3>Project 1</h3>
+					      <p>Lorem ipsum dolor amet flexitarian lomo intelligentsia</p>
+					    </div>
+					  </div>
+					
+					  <div class="column">
+					    <div class="content">
+					     <img src="Images/pholder.jpg" alt="placeholder" style="width: 100%">
+					      <h3>Project 2</h3>
+						  <p>Authentic put a bird on it, pour-over celiac mlkshk</p>
+					    </div>
+					  </div>
 				
-				<!--
-				<section class="portfolio">
-													<div class="container">
-														<h1>{Portfolio}</h1>
-													</div>
-								</section>	
-								
-								<section class="contact">
-													<div class="container">
-														<h1>{Contact Me}</h1>
-													</div>-->
-						
-						<!--
-						</section>
+					  <div class="column">
+						    <div class="content">
+						      <img src="Images/pholder.jpg" alt="placeholder" style="width: 100%">
+						      <h3>Project 3</h3>
+						      <p>Taxidermy. Hell of master cleanse tumeric lo-fi</p>
+						    </div>
+						  </div>
+					
+				      <div class="column">
+						      <div class="content">
+						        <img src="Images/pholder.jpg" alt="placeholder" style="width:100%">
+						        <h3>Project 4</h3>
+						        <p>Pok pok fixie try-hard VHS scenester man braid</p>
+						      </div>
+						    </div>
+		
+					  <div class="column">
+							    <div class="content">
+							      <img src="Images/pholder.jpg" alt="placeholder" style="width:100%">
+							      <h3>Project 5</h3>
+							      <p>Chillwave hashtag four dollar toast neutra. Disrupt</p>
+							    </div>
+							  </div>
+			
+					  <div class="column">
+							      <div class="content">
+							        <img src="Images/pholder.jpg" alt="placeholder" style="width:100%">
+							        <h3>Project 6</h3>
+							        <p>Umami truffaut you probably haven't heard of them</p>
+							      </div>
+							    </div>
+							</div>
+		</section>	
 								
-								<section class="container">
-										<footer>
-											<p>Matt Martin &copy2020</p>
-										</footer>
-								</section>
-						-->
-				
-				
+		
```

Comment:
I think there may be an unclosed tag somewhere but hard to tell until you have corrected your indentation.